src/codegeneration/ModuleNormalizeTransformer.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2015 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  AnonBlock,
+  ImportDeclaration,
+  ImportSpecifier,
+  ImportSpecifierSet,
+  Module,
+} from '../syntax/trees/ParseTrees.js';
+import {ParseTreeTransformer} from './ParseTreeTransformer.js';
+import {
+  ANON_BLOCK,
+  IMPORT_CLAUSE_PAIR,
+  IMPORT_DECLARATION,
+  IMPORT_SPECIFIER_SET,
+  IMPORTED_BINDING,
+  NAME_SPACE_IMPORT,
+} from '../syntax/trees/ParseTreeType.js';
+import {createIdentifierToken} from './ParseTreeFactory.js';
+
+/**
+ * Normalizes import declarations to a simpler form. This is so that the
+ * ModuleTransformer only has to handle the basic import declarations.
+ */
+export class ModuleNormalizeTransformer extends ParseTreeTransformer {
```

Comment:
`ImportSimplifyingTransformer`? `ImportNormalizeTransformer`?  It doesn't seem to normalize a module.